gulpfile.js

```diff
@@ -764,6 +765,17 @@ gulp.task('check-deploy', ['build-docs'], function() {
   });
 });
 
+gulp.task('firebase-use-proj-check', cb => {
+  try {
+    execSync('firebase use');
+  } catch (e) {
+    // Rerun command so user gets project + alias info
+    execSync('firebase use', {stdio:[0,1,2]});
+    throw `\nAborting: no firebase project selected. Run:\n\n  firebase use <project-or-alias-name>\n\n`;
```

Comment:
Perhaps a little help on how to get the project-or-alias-name? Or an example of a name?